backend/pyproject.toml

```diff
@@ -25,7 +25,7 @@ Flask = ">=1.0.2"
 werkzeug  = ">=0.16.1,<1.0.0"
 msgpack = "==0.6.2"
 
-python = ">=3.6,<3.8"
+python = ">=3.6,<3.9"
```

Comment:
don't you want to require at least 3.8 now? if we run all the CI and deployments on 3.8, we better make sure that the developers are using the same version, otherwise things might start failing locally when incompatible with older versions.